draft-reddy-tls-composite-mldsa.md

```diff
@@ -156,19 +161,33 @@ to be taken into account.
 This document requests new entries to the TLS SignatureScheme registry,
 according to the procedures in {{Section 6 of TLSIANA}}.
 
+
 | Value   | Description                         | Recommended | Reference      |
 |---------|-------------------------------------|-------------|----------------|
 | 0x0907  | mldsa44_ecdsa_secp256r1_sha256      | N           | This document. |
-| 0x0908  | mldsa65_ecdsa_secp384r1_sha384      | N           | This document. |
-| 0x0909  | mldsa87_ecdsa_secp384r1_sha384      | N           | This document. |
+| 0x0908  | mldsa65_ecdsa_secp384r1_sha512      | N           | This document. |
+| 0x0909  | mldsa87_ecdsa_secp384r1_sha512      | N           | This document. |
 | 0x090A  | mldsa44_ed25519                     | N           | This document. |
 | 0x090B  | mldsa65_ed25519                     | N           | This document. |
-| 0x090C  | mldsa44_rsa_pkcs1_sha256            | N           | This document. |
-| 0x090D  | mldsa65_rsa_pkcs1_sha384            | N           | This document. |
-| 0x090E  | mldsa44_rsa_pss_pss_sha256          | N           | This document. |
-| 0x090F  | mldsa65_rsa_pss_pss_sha384          | N           | This document. |
+| 0x090C  | mldsa44_rsa3072_pkcs1_sha256        | N           | This document. |
+| 0x090D  | mldsa65_rsa4096_pkcs1_sha384        | N           | This document. |
+| 0x090E  | mldsa44_rsa3072_pss_pss_sha256      | N           | This document. |
+| 0x090F  | mldsa65_rsa4096_pss_pss_sha384      | N           | This document. |
```

Comment:
I think that list should match what is there in [draft-ietf-lamps-pq-composite-sigs-03](https://datatracker.ietf.org/doc/html/draft-ietf-lamps-pq-composite-sigs-03) so that peers can clearly indicate which hybrids they support from that draft.

I don't see algorithms that match:
 * id-MLDSA44-RSA2048-PSS
 * id-MLDSA44-RSA2048-PKCS15
 * id-MLDSA65-RSA3072-PSS
 * id-MLDSA65-RSA3072-PKCS15
 * id-MLDSA65-ECDSA-brainpoolP256r1
 * id-MLDSA87-ECDSA-brainpoolP384r1
 * id-MLDSA87-Ed448

And I don't see algorithms that match
* mldsa44_rsa3072_pss_pss_sha256 
* mldsa44_rsa3072_pss_pss_sha256

Reply:
a) brainpoolP256r1 and brainpoolP384r1 are not included as it does meet the selection criteria for composite signature algorithms as discussed in https://www.ietf.org/archive/id/draft-reddy-tls-composite-mldsa-00.html#section-3. 

b) MLDSA87-Ed448 is added to the signature scheme, please re-check.

c) Updated the draft to include the missing algorithms and removed the not matching ones, please check.

Reply:
ok, looks good now